docs/readmes/howtos/call_tracing.md

```diff
@@ -0,0 +1,65 @@
+---
+id: call_tracing 
+title: Call Tracing
+hide_title: true
+---
+
+*Last Updated: 1/20/2021*
+
+# Call Tracing
+
+Magma supports basic traffic capture for troubleshooting purposes.
+Monitoring of control messaging flow and other traffic between the Magma access
+gateway and eNodeB devices is possible with this feature.
+
+As of the time of writing, we are in the process of adding filtering for
+specific protocols and allowing custom options through tshark.
+
+### Requirements
+
+Ensure you have the following:
+
+* a functional orc8r
+* a configured LTE network
+* a configured LTE gateway with eNodeB
+* subscribers that can attach to your LTE gateway
+* network running in un-federated mode
+
+### Initiating a Call Trace
```

Comment:
For the sake of completion, can you add a API reference section and add all the relevant APIs which can be used to control call tracing as well